### DISCUSSION TITLE
Setup in Voltage: Please note that creating a hot wallet is not supported by this instance for non administrators?

### DISCUSSION THREAD

**Discussion by olimits7** [btcpayserver/btcpayserver#4073] [Technical Support]
Hi,

I set up BTCPay Server under my Voltage account but when I try to create a hot wallet it's grayed out with this message "**Please note that creating a hot wallet is not supported by this instance for non administrators.**" showing and I'm logged in as an "**Owner**".

Why am I not able to create a hot wallet?

Thank you!

**Comment by Kukks** [comment]
Voltage configured their server that way, you need to ask them.

**Reply by olimits7** [reply]
Ok, ty...I'll reach out to Voltage to see what the issue is.

**Reply by ziziele** [reply]
> Ok, ty...I'll reach out to Voltage to see what the issue is.

Do you know what the issue is? I have the same problem.

**Reply by olimits7** [reply]
This is the reply I received from Voltage regarding hot wallets:

> All of Voltage's products are non-custodial, which means we do not want to have the ability to send your funds. Allowing the Hot Wallet feature would violate that design.
 
In my opinion, an even better (and safer) setup than using the Hot Wallet feature is to use the "Connect Existing Wallet" options. From there, you have a few different ways to upload PUBLIC KEYS (something that lets you generate addresses but not send funds). I recommend creating a wallet in Sparrow or Electrum, and then copying the XPUB (extended public key) from that wallet into your BTCPayServer setup. Make sure to back up your wallet seeds, whichever way you prefer, ideally in metal, depending on how much funds you plan to have on that wallet.
